polybar-scripts/info-tmux-sessions/info-tmux-sessions.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
Is there a reason why we should not write a shell independent script with `#!/bin/sh`?
The most of the scripts in this repo are that. We could work on this.

Reply:
I suppose I did this out of habbit.

Also, I'm not sure that posix would like how I check for the exit value 1 that comes back when there are no sessions.

I am willing to change it to /bin/sh.

I guess it makes my disclaimer that I've only tested it on the latest version of bash all the more important.

Reply:
I would love to see this and other users too I guess :sunglasses: 
Write a comment if you need help.

Reply:
Oh, you mean have it tested on sh ?

My Arch system has a symlink to /bin/bash located at /bin/sh.  I might be able to test this on a server that isn't Arch though.

Reply:
I have now tested it with the `--posix` flag set, and all seems well.

I do not know if this is rigorous enough testing.  Let me know if you have extra ideas for verifying it's compatibility with other shells.

I'll commit an update to the shebang when I've made the new screenshots.

Reply:
I have confirmed this does not work with dash.

Reply:
dash doesn't have `[[`

Reply:
After some more research I believe this will work with the more advanced members of the korn shell family, but I do not expect this to work with the plain Bourne shell.

Reply:
It is possible to rework it for compatibility with dash, etc.. , but it would involve external commands like `sed`.

Overall it would be harder to read, and probably slower due to the workarounds and external programs.  Right now it takes less than 1/100 of a second on my machine.  Should I add this as a disclaimer?  People on debian will get dash if it is invoked with `/bin/sh`, and then it will be broken for them.

Reply:
Also a problem for void users.

Reply:
You can use `shellcheck` to see what happens. 
The main problem is that you use arrays. POSIX can't handle arrays. The qestion would be: Is it really needed?